vite.config.js

```diff
@@ -1,5 +1,99 @@
 import { sveltekit } from '@sveltejs/kit/vite';
 import svg from '@poppanator/sveltekit-svg';
+import { normalizePath } from 'vite';
+import { relative, resolve, basename, join } from 'path';
+import picomatch from 'picomatch';
+import colors from 'picocolors';
+import * as esbuild from 'esbuild';
+import { writeFile } from 'fs/promises';
+
+/**
+ * Normalize multiple paths
+ * @param {string} root Root path to normalize relative to
+ * @param {string | string[]} path The paths to normalize
+ * @returns {string[]}
+ */
+export function normalizePaths(root, path) {
+  return (Array.isArray(path) ? path : [path]).map(path => resolve(root, path)).map(normalizePath);
+}
+
+/**
+ * The Config object for the Worker
+ *
+ * @typedef {Object} WorkerBuildConfig
+ * @property {string | undefined} outdir The directory to send built files, defaults to `./static`
+ * @property {boolean | undefined} minify If we should minify built files, defaults to the setting in the overall vite configuration.
+ * @property {boolean | undefined} log If we should log when files change. Defaults to true
+ */
+
+/** A quick and dirty Vite plugin to hook in the esbuild rebuild
+ * @param {string[]} paths The file paths we want to watch. Does not support globbing, so specify all files exactly!
+ * @param {WorkerBuildConfig} config
+ * @returns {import('vite').PluginOption}
+ */
+const WorkerBuildPlugin = (paths, config) => ({
```

Comment:
To keep the `vite.config.js` cleaner, can we export this `WorkerBuildPlugin` from it's own file?